src/sentry/api/endpoints/project_rules.py

```diff
@@ -73,23 +81,67 @@ def find_duplicate_rule(rule_data, project, rule_id=None):
     return None
 
 
+class ProjectRulesPostSerializer(serializers.Serializer):
+    name = GlobalParams.name("The name for the rule.")
+    conditions = serializers.ListField(
+        child=serializers.DictField(),
+        help_text="A list of triggers that determine when the rule fires.",
+    )
+    filters = serializers.ListField(
+        child=serializers.DictField(),
+        required=False,
+        help_text="A list of filters that determine if a rule fires after the listed conditions have been met.",
+    )
+    filterMatch = serializers.ChoiceField(
+        choices=(("all", "all"), ("any", "any"), ("none", "none")),
+        required=False,
+        help_text="An operator determining which filters need to hold before any actions take place. Required when `filters` is passed in.",
+    )
+    actions = serializers.ListField(
+        child=serializers.DictField(),
+        help_text="A list of actions that will occur when all required conditions and filters for the rule are met.",
+    )
+    actionMatch = serializers.ChoiceField(
+        choices=(("all", "all"), ("any", "any"), ("none", "none")),
+        help_text="An operator determining which actions should take place when the rule triggers.",
+    )
+    environment = GlobalParams.ENVIRONMENT
+    frequency = serializers.IntegerField(
+        min_value=5,
+        max_value=60 * 24 * 30,
+        help_text="How often the alert rule can be triggered for a particular issue, in seconds.",
+    )
+    owner = serializers.IntegerField(
+        required=False, allow_null=True, help_text="The ID of the team or user that owns the rule."
+    )
```

Comment:
could not use `RuleNodeField` for conditions/filters/actions nor `ActorField` for owner here, so simplified them